player/events/events_test.go

```diff
@@ -0,0 +1,16 @@
+package events
+
+import (
+	"testing"
+
+	"github.com/luno/reflex/rsql"
+
+	"bitx/console/ticketing/internal/db"
```

Comment:
wrong import, we will have a db/connect.go from sergio soon for db.ConnectFor...()

Reply:
oops